recipes/libpq/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake
+from conan.tools.build import cross_building
+import os
+
+
+class TestPackageV1Conan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package"
```

Reply:
I don't think this is the issue. Local tests are failing with the same error